Symfony/CS/Fixer/PSR2/OneLineMethodArgumentFixer.php

```diff
@@ -0,0 +1,208 @@
+                if (!$this->isNewLine($tokens[$index - 1]) && $tokens[$index - 1]->getContent() !== '(') {
+                    $this->addNewLineOn($tokens, $index - 1, $endFunctionIndex, '');
+                }
+            }
+
+            if ($token->equals(',')) {
+                $this->fixNewLine($tokens, $index, $endFunctionIndex);
+            }
+        }
+    }
+
+    /**
+     * Inserts new line after comma.
+     *
+     * @param Tokens $tokens
+     * @param int    $index
+     * @param int    $endFunctionIndex
+     */
+    private function fixNewLine(Tokens $tokens, $index, &$endFunctionIndex)
+    {
+        if (
+            $tokens[$index + 1]->isWhitespace()
+            && $tokens[$index + 2]->isComment()
+            && $this->isNewLine($tokens[$index + 3])
+        ) {
+            $index += 3;
+        }
+
+        $this->addNewLineOn($tokens, $index, $endFunctionIndex);
+    }
+
+    /**
+     * Adds new line to the given token.
+     *
+     * @param Tokens $tokens
+     * @param int    $index
+     * @param int    $endFunctionIndex
+     * @param string $indention
+     */
+    private function addNewLineOn(Tokens $tokens, $index, &$endFunctionIndex, $indention = '    ')
+    {
+        $newToken = new Token(array(T_WHITESPACE, "\n".$indention));
+
+        if ($tokens[$index]->isWhitespace()) {
+            $tokens->overrideAt($index, $newToken);
+        } else {
+            if ($tokens[$index + 1]->isWhitespace()) {
+                $tokens->overrideAt($index + 1, $newToken);
+            } else {
+                $tokens->insertAt($index + 1, $newToken);
+                $endFunctionIndex++;
+            }
+        }
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $startIndex
+     * @param int    $endIndex
+     *
+     * @return bool
+     */
+    private function hasNewLineBetweenParenthesis(Tokens $tokens, $startIndex, $endIndex)
+    {
+        /** @var Token $token */
+        foreach ($tokens as $index => $token) {
+            if ($index <= $startIndex || $index >= $endIndex) {
+                continue;
+            }
+            if ($this->isNewLine($token)) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return int|null
+     */
+    private function getFunctionCloseParenthesisIndex(Tokens $tokens, $index)
+    {
+        $tokenIndex = $tokens->getNextNonWhitespace($index);
+
+        return $tokens[$tokenIndex]->getContent() === ')' && $tokens[$tokenIndex - 1]->getContent() !== '('
+            ? $tokenIndex
+            : $this->getFunctionCloseParenthesisIndex($tokens, $tokenIndex);
+    }
+
+    /**
+     * Checks if token is new line.
+     *
+     * @param Token $token
+     *
+     * @return bool
+     */
+    private function isNewLine(Token $token)
+    {
+        return trim($token->getContent(), " \r\t\0\x0B") === "\n";
```

Comment:
you won't need to trim every single kind of token
first, check if token is one of kind that may be multiline.